notification_configuration_integration_test.go

```diff
@@ -34,13 +34,16 @@ func TestNotificationConfigurationList(t *testing.T) {
 		assert.Contains(t, ncl.Items, ncTest1)
 		assert.Contains(t, ncl.Items, ncTest2)
 
-		t.Skip("paging not supported yet in API")
```

Comment:
Removed the skips for these tests. They work even though we don't really support pagination for these endpoints (we cap at 20 notification configurations maximum per subscribable).